.github/workflows/test-e2e-ios.yml

```diff
@@ -49,9 +43,16 @@ jobs:
         run: detox clean-framework-cache && detox build-framework-cache
       - name: Build Detox-version of app
         if: steps.detox-app-cache.outputs.cache-hit != 'true'
-        run: detox build --configuration ${{ inputs.e2econfig }}
+        run: detox build --configuration ios.release
       - name: Replace bundle in app cache
         if: steps.detox-app-cache.outputs.cache-hit == 'true'
         run: npx react-native bundle --platform ios --dev false --reset-cache --entry-file index.js --bundle-output main.jsbundle && mv main.jsbundle build/detox/ios/Build/Products/Release-iphonesimulator/AtB.app/
       - name: Run tests
-        run: detox test --configuration ${{ inputs.e2econfig }} --loglevel trace
+        run: detox test --configuration ios.release --loglevel trace --retries 3 --take-screenshots failing
```

Comment:
--retries 3
Betyr dette at den tar hele "describe" om igjen, eller tar den kun den ene "it" som feiler?

Reply:
👍  Enig. 1 eller 2.